cmd/ctr/snapshot.go

```diff
@@ -274,6 +275,36 @@ var treeSnapshotCommand = cli.Command{
 	},
 }
 
+var infoSnapshotCommand = cli.Command{
+	Name:      "info",
+	Usage:     "get info about a snapshot",
+	ArgsUsage: "<key>",
+	Action: func(clicontext *cli.Context) error {
+		ctx, cancel := appContext(clicontext)
+		defer cancel()
+
+		if clicontext.NArg() != 1 {
+			return cli.ShowSubcommandHelp(clicontext)
+		}
+
+		key := clicontext.Args().Get(0)
+
+		snapshotter, err := getSnapshotter(clicontext)
+		if err != nil {
+			return err
+		}
+
+		info, err := snapshotter.Stat(ctx, key)
+		if err != nil {
+			return err
+		}
+
+		printInfo(info)
```

Comment:
Let's get rid of this extra function, the other objects just call `printAsJSON` directly